iOS/MaterialComponents/source/MaterialComponents/Extensions.cs

```diff
@@ -93,6 +93,30 @@ partial class BottomAppBarColorThemer {
 		}
 	}
 
+	partial class Card {
+		public void SetShadowElevation (double shadowElevation, UIControlState state) {
```

Comment:
I would suggest adding the method with a `Wrap` attribute below the original signature

```csharp
[Wrap ("SetShadowElevation ((nfloat)shadowElevation, state)")]
void SetShadowElevation (double shadowElevation, UIControlState state);
```

But that's my feel. Either way, LGTM. 👍🏼